crm_partner_schedule_meeting/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- encoding: utf-8 -*-
```

Comment:
I think this file does not belong here. It makes Odoo crash. Solved deleting the folder.